src/activities/AddImageToPdf.ts

```diff
@@ -0,0 +1,154 @@
+import type { IActivityHandler } from "@vertigis/workflow";
+import { ColorTypes, PDFDocument, PDFImage } from "pdf-lib";
+import { hexToRgba } from "./pdfUtils";
+
+interface AddImageToPdfInputs {
+    /**
+     * @description The source PDF document to modify.
+     * @required
+     */
+    source: Blob;
+
+    /**
+     * @description The image to add. Only JPEG and PNG images are supported.
+     * @required
+     */
+    image: Blob | ArrayBuffer;
+
+    /**
+     * @description The X coordinate (in points) to add the image at. The coordinate [0, 0] represents the bottom left corner of the page.
+     * @required
+     */
+    x: number;
+
+    /**
+     * @description The Y coordinate (in points) to add the image at. The coordinate [0, 0] represents the bottom left corner of the page.
+     * @required
+     */
+    y: number;
+
+    /**
+     * @description The height of the image.
+     */
+    height: number;
+
+    /**
+     * @description The width of the image.
+     */
+    width: number;
+
+    /**
+     * @description The width (in points) of the image border. The default is 0 (no border).
+     */
+    borderWidth?: number;
+
+    /**
+     * @description The RGBA hex color of the image border. The default is 000000FF (black). The alpha value is optional.
+     */
+    borderColor?: "000000FF" | "FF0000FF" | "0000FFFF" | string;
+
+    /**
+     * @description The zero-based index of the page to add the image to. The default is 0.
+     */
+    pageIndex?: number;
+}
+
+interface AddImageToPdfOutputs {
+    /**
+     * @description The modified PDF document.
+     */
+    result: Blob;
+}
+
+/**
+ * @displayName Add Image To PDF
+ * @category PDF
+ * @description Adds an image to a PDF document.
+ * @clientOnly
+ * @supportedApps EXB, GWV, GVH, WAB
```

Comment:
Should we start using `VSW` instead of `GWV` in these? 5.36 can understand both and the activity packs won't be listed in older designers. On the other hand, someone could manually register this activity pack and use an older version of the on-prem designer. If that was the case, `GWV` would be required.

Reply:
If we do then we'll have to mark this PR as a breaking change and update the requirements to say 5.36.

This activity pack has been available for a couple months, but we have no way to verify if anyone has registered it manually. It wouldn't be many.

I think we might want to make that sort of change across all 15 activity packs at the same time to avoid confusion. We'd probably need to wait for another year for that to ensure significant adoption of 5.36+.